engine/installation/linux/docker-ee/suse.md

```diff
@@ -95,6 +118,9 @@ BTRFS filesystem and mount it on `/var/lib/docker/`:
     Don't forget to make the change permanent across reboots by adding an
     entry to `/etc/fstab`.
 
+4.  If `/var/lib/docker` previously existed and you backed up its contents
+    during step 1, restore them onto `/var/lib/docker`.
```

Comment:
restore it

Reply:
"contents" are plural so I think it is "them".